shell/packages/sandstorm-permissions/permissions-tests.js

```diff
@@ -0,0 +1,137 @@
+// Sandstorm - Personal Cloud Sandbox
+// Copyright (c) 2015 Sandstorm Development Group, Inc. and contributors
+// All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+var globalDb = new SandstormDb();
+
+globalDb.collections.grains.remove({});
```

Comment:
// TODO(cleanup): Use mock DB.

(I think a mock DB should be easy to implement on top of minimongo. Basically just pass null for all of the collection names.)